pages/makes.jsx

```diff
@@ -106,8 +106,9 @@ var MePage = React.createClass({
     if (!this.state.username) {
       pageContent = <span>Please sign in.</span>;
     } else if (this.state.loadingMakes) {
-      pageContent = <span>Loading Makes...</span>;
-    } else {
+      pageContent = <div className="loading-message">Loading makes</div>;
```

Comment:
do we still use the term "makes" to refer to things published using the **new tools**? if not, I think calling these "makes" is fine :eyes: